b2share/modules/records/cli.py

```diff
@@ -102,6 +104,49 @@ def check_and_update_handle_records(update, verbose):
                         click.secho('  file PID ok: {}'.format(pid))
 
 
+@b2records.command()
+@with_appcontext
+@click.option('-u', '--update', is_flag=True, default=False)
+@click.argument('record_id', required=True)
+def check_handles(update, record_id):
```

Comment:
Do we want to keep `check_and_update_handle_records` above this function? Both functions check the PIDs, seems redundant to me.

Reply:
`check_and_update_handle_records` adds the EUDAT/xx entries to pid handles, it has a different purpose. `check_handles` adds missing pids to a record and its files.